arch/RISCV/RISCVGenAsmWriter.inc

```diff
@@ -2559,13 +2562,22 @@ static bool printAliasInstr(MCInst *MI, SStream * OS, void *info)
     return false;
   }
 
-  tmpString = cs_strdup(AsmString);
+  AsmStringLen = strlen(AsmString);
+  if (ASMSTRING_CONTAIN_SIZE - 1 < AsmStringLen)
+    tmpString = cs_strdup(AsmString);
+  else
+    tmpString = memcpy(tmpString, AsmString, 1 + AsmStringLen);
```

Comment:
if you are quite sure about 64 bytes is enough, how about just use strncpy() to copy, and avoid cs_strdup() totally?

Reply:
I am not sure the `strlen(AsmString)` will exceed `64` or not in the feature.

Reply:
or else I can change `cs_strdup` to `assert`?

Reply:
if you cannot be sure, then just use heap memory, otherwise i dont see any point of mixing stack variable with heap memory like this.

anyway i guess you can be fairly sure that instruction mnemonic cannot be ever longer than (lets say) 64 bytes, which is way too long instruction name.

Reply:
for now, 64 is enough. `cs_strdup` is a safeguard.

Reply:
instruction mnemonic of RISCV is unlikely ever to be that long. you can look at Intel, which has far more instructions, but the instruction size is still less than 20 i thinks.

FYI, 32 is the current upper size limit of Capstone mnemonic for all archs: https://github.com/aquynh/capstone/blob/master/include/capstone/capstone.h#L68

Reply:
I know, for the current version, `cs_strdup()` will never be called.
I write it here mainly for the condition test complete and safeguard.